virttest/tests/unattended_install.py

```diff
@@ -399,6 +399,16 @@ def answer_kickstart(self, answer_path):
             raise ValueError("Unexpected installation medium %s" % self.url)
         contents = re.sub(dummy_medium_re, content, contents)
 
+        dummy_guest_name_re = r'\bRH_GUEST_NAME\b'
+        if re.search(dummy_guest_name_re, contents):
+            guest_name = self.params.get("guest_name")
+            contents = re.sub(dummy_guest_name_re, guest_name, contents)
```

Comment:
Won't this error out if `guest_name` is not defined?

Reply:
@pevogam It has to be defined together with `dummy_guest_name_re`, so it is intended to raise error if it is not, but should not be here, the above line should be `guest_name = self.params["guest_name"]`, @6-dehan

Reply:
@pevogam @vivianQizhu IMO, it is not must to define `guest_name` together with `dummy_guest_name_re`. Miss `guest_name` should not effect the test.   But you are right, the code need be updated, i suggest `params.get("guest_name", '')`.  And i suggest to use one param to cover this as the pre comment, that is another story ~

1. No `guest_name`, but `dummy_guest_name_re` in the `content`, it will be replaced with '',  and some test was expected to be skipped in ks.
2. No `dummy_guest_name_re` in the `content`, but `guest_name` was specified, it will not take effect in the ks, nothing ~

Reply:
@yanan-fu Yes you are right.

Reply:
Using `params.get("guest_name", '')` sounds much better indeed. And using more specific parameter name than `guest_name` is definitely welcome too.